components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/bootstrap-editable.js

```diff
@@ -0,0 +1,6807 @@
+/*! X-editable - v1.5.1 
```

Comment:
Don't copy libraries again and again in the fragments. Its better to have libraries in a common place. This js file already included  in [1]
[1] - https://github.com/wso2/carbon-apimgt/blob/C5/components/web/themes/org.wso2.carbon.apimgt.theme.publisher/src/main/public/lib/bootstrap3-editable/js/bootstrap-editable.js